modules/codemod/lib/v7/recategorizeIconButtons.ts

```diff
@@ -69,11 +70,19 @@ export default function transformer(file: FileInfo, api: API, options: Options)
 
       const variantProp = attrs?.find(
         attr => attr.type === 'JSXAttribute' && attr.name.name === 'variant'
-      );
+      ) as JSXAttribute | undefined;
 
       // Default IconButton variant is `circle`
       if (variantProp) {
-        const variantPropValue = ((variantProp as JSXAttribute).value as StringLiteral)?.value;
+        const valueBody = variantProp.value;
+        if (valueBody?.type === 'JSXExpressionContainer') {
+          variantProp.value = valueBody.expression as StringLiteral;
+        }
+
+        const variantPropValue = (variantProp.value as StringLiteral).value;
+
+        (variantProp.value as StringLiteral).value = variantPropValue.replace('Filled', '');
```

Comment:
Thank you for fixing this! @jamesfan this fixes the issues you mention of the `inverseFilled` incorrectly mapping